libs/blocks/tabs/tabs.css

```diff
@@ -172,7 +180,35 @@
   margin-inline-start: 16px;
 }
 
-.tabs.quiet div[role="tablist"] button:first-of-type {
+.tabs[class*='pill'] {
```

Comment:
I'm just curious why are you using this approach instead of `.tabs.pill`.
The following element will match your selectors, it might create some weird side-effects in the future.
```
<div class="tabs pillow"></div>
```

Reply:
@narcis-radu the reason we are using that selector is because the `pill` variant can have sizes `pill-small` and `pill-large`